views/courses/create-course.hbs

```diff
@@ -45,7 +45,7 @@
                 {{$t "courses.add.tab_label.finish"}}
               </div>
             </div>
-          {{#embed "lib/forms/paginatedForm" sections=3 action=action method=method submit-label=submitLabel}}
+          {{#embed "lib/forms/paginatedForm" sections=3 action=action method=method submit-label=submitLabel submit-testid=submitTestId}}
```

Comment:
is submit-testid the correct parameter to hand this parameter over? is that used in the controller to hand that parameter? did not find that usage somewhere else in code so just asking for reassurance. especially as deleteCourse (below) uses "data-testid". also why is camel case used here instead of the other usages of "*_*"?

Reply:
Introduced property submit-testid to paginatedForm.hbs in order to enable passing a testId for the submit button into the paginated form. changed the passed id to use "_".